cobalt/renderer/backend/egl/framebuffer.cc

```diff
@@ -12,15 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "starboard/configuration.h"
-
 #include "cobalt/renderer/backend/egl/framebuffer.h"
 
 #include "base/logging.h"
 #include "cobalt/renderer/backend/egl/graphics_context.h"
 #include "cobalt/renderer/backend/egl/texture.h"
 #include "cobalt/renderer/backend/egl/utils.h"
 #include "cobalt/renderer/egl_and_gles.h"
+#include "starboard/configuration.h"
```

Comment:
These are a little scary, because they could highlight missing includes of this file in any of the dependent includes. I didn't see any in a spot check, but that doesn't mean I checked everywhere.

Reply:
I actually ran into missing includes in a few places with those, but as long as all platforms build and pass tests, i don't expect further problems from those.